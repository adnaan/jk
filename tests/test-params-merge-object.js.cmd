jk run --parameters %t.json --po myObject.o={"xxx":"yyy"} %f
