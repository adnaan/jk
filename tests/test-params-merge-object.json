{
  "myBoolean": true,
  "myNumber": 2,
  "myObject": {
    "message": "success",
    "o": {
      "foo": "bar"
    }
  },
  "myString": "success"
}
