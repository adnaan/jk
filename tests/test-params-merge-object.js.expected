{
  "myBoolean": true,
  "myNumber": 2,
  "myObject": {
    "message": "success",
    "o": {
      "foo": "bar",
      "xxx": "yyy"
    }
  },
  "myString": "success"
}
